h2o-py/h2o/model/model_base.py

```diff
@@ -1401,3 +1501,20 @@ def _get_matplotlib_pyplot(server):
     except ImportError:
         print("`matplotlib` library is required for this function!")
         return None
+
+def _get_mplot3d_pyplot(functionName):
```

Comment:
Typically private function in Python should start with `__`

Reply:
Yeap, fixed! Thanks.